src/test/scala/org/broadinstitute/dsde/workbench/sam/google/DisableUsersMonitorSpec.scala

```diff
@@ -0,0 +1,92 @@
+package org.broadinstitute.dsde.workbench.sam.google
```

Comment:
Would it be possible/valuable to add a test that goes through the `FailToDisable` path? It doesn't seem like that path will really happen since `disableUser` always seems to return a successful Future, but I'm not positive

Reply:
I don't think it will ever go through that path, since either `disableUser` will either return a successful future or a none. I could add a test which throws an exception just in case we add some exceptions to the endpoint? But that feels future-proof-y, and not necessarily correct for current behavior.